src/index.tsx

```diff
@@ -1,17 +1,21 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom";
+import { CssBaseline } from "@mui/material";
+import { QueryClient, QueryClientProvider } from "react-query";
+import { ReactQueryDevtools } from "react-query/devtools";
+import App from "./components/App/App";
```

Comment:
./components/App/App could use index.tsx file there and shorten this import to ./components/App